features/step_definitions/tournament.rb

```diff
@@ -41,10 +41,14 @@
   visit event_schedule_path(tournament_named tournamentName)
 end
 
-And(/^the texts? (.+) (?:are|is) there\.$/) do |texts_raw|
+And(/^the texts? (.+) (?:are|is)( not)? there\.?$/) do |texts_raw, _not|
```

Comment:
why `_not`?

Reply:
because `<space>not` and `not` are not valid variable names

Reply:
see comment below